index.d.ts

```diff
@@ -1367,14 +1367,44 @@ declare namespace Shopify {
         code: string;
         type: OrderDiscountCodeType
     }
+    
+    interface IOrderFulfillmentLineItemTaxLine {
+        title: string;
+        price: string;
+        rate: number;
+    }
+    
+    interface IOrderFulfillmentLineItem {
+        id: number;
+        variant_id: number;
+        title: string;
+        quantity: number;
+        price: string;
+        grams: number;
+        sku: string;
+        variant_title: string;
+        vendor: string | null;
+        fulfillment_service: string;
+        product_id: number;
+        requires_shipping: boolean;
+        taxable: boolean;
+        gift_card: boolean;
+        name: string;
+        variant_inventory_management: string;
+        product_exists: boolean;
```

Comment:
The `properties` property is missing.

Reply:
Fixed by https://github.com/MONEI/Shopify-api-node/pull/250/commits/a877d03b6d964e6ec02d7e8043efa887958d232d